haskell-language-server.cabal

```
@@ -124,54 +123,55 @@ common cabalfmt
 flag isolateCabalfmtTests
   description: Should tests search for 'cabal-fmt' on the $PATH or shall we install it via build-tool-depends?
   -- By default, search on the PATH
-  default:     False
-  manual:      True
+  default: False
+  manual: True
 
 library hls-cabal-fmt-plugin
-  import:           defaults, pedantic, warnings
+  import: defaults, pedantic, warnings
+
   if !flag(cabalfmt) || !flag(cabal)
     buildable: False
-  exposed-modules:  Ide.Plugin.CabalFmt
-  hs-source-dirs:   plugins/hls-cabal-fmt-plugin/src
-  build-depends:
-    , directory
-    , filepath
-    , ghcide          == 2.11.0.0
-    , hls-plugin-api  == 2.11.0.0
-    , lens
-    , lsp-types
-    , mtl
-    , process-extras
-    , text
+  exposed-modules: Ide.Plugin.CabalFmt
+  hs-source-dirs: plugins/hls-cabal-fmt-plugin/src
+  build-depends:
+    directory,
+    filepath,
+    ghcide ==2.11.0.0,
+    hls-plugin-api ==2.11.0.0,
+    lens,
+    lsp-types,
+    mtl,
+    process-extras,
+    text,
 
 -- The `hls-cabal-plugin` is needed for tests, as we need to install notification handlers
 test-suite hls-cabal-fmt-plugin-tests
-  import:           defaults, pedantic, test-defaults, warnings
+  import: defaults, pedantic, test-defaults, warnings
+
   if !flag(cabalfmt) || !flag(cabal)
     buildable: False
-  type:             exitcode-stdio-1.0
-  hs-source-dirs:   plugins/hls-cabal-fmt-plugin/test
-  main-is:          Main.hs
-  build-depends:
-    , directory
-    , filepath
-    , haskell-language-server:hls-cabal-plugin
-    , haskell-language-server:hls-cabal-fmt-plugin
-    , hls-plugin-api        == 2.11.0.0
-    , hls-test-utils        == 2.11.0.0
-
-  if flag(isolateCabalfmtTests)
+  type: exitcode-stdio-1.0
+  hs-source-dirs: plugins/hls-cabal-fmt-plugin/test
+  main-is: Main.hs
+  build-depends:
+    directory,
+    filepath,
+    haskell-language-server:hls-cabal-fmt-plugin,
+    haskell-language-server:hls-cabal-plugin,
```

Just so you know, these can use set notation:
```suggestion
    haskell-language-server:{hls-cabal-fmt-plugin, hls-cabal-plugin},
```